lib/aff4.py

```diff
@@ -2496,13 +2498,18 @@ def _ReadPartial(self, length):
 
     return result
 
-  def Read(self, length):
+  def Read(self, length=None):
     """Read a block of data from the file."""
     result = ""
-
+    maxread = config_lib.CONFIG["Server.max_unbound_read_size"]
```

Comment:
This should be calculated once in the constructor because this might slow down Read() calls for all objects. The config lib might have to do some calculations to extract this parameter (its not just a dict lookup).